Composer/packages/tools/language-servers/language-generation/src/utils.ts

```diff
@@ -125,3 +132,8 @@ export function checkTemplate(template: Template): LGDiagnostic[] {
     return diagnostic.message.includes('does not have an evaluator') === false;
   });
 }
+
+export type MemoryVaribleCompletionResult = {
+  endWithDotFlag: boolean;
```

Comment:
endWithDot is good enough

and what's this flag used for?

Reply:
This flag looks like a global state, but here is a local state in the `result`, looks weird